hbt/config/configs_hbt.py

```diff
@@ -147,6 +147,10 @@ def if_era(
         if process_name.startswith(("graviton_hh_", "radion_hh_")):
             proc.add_tag("signal")
             proc.add_tag("resonant_signal")
+        if process_name.startswith("tt"):
+            proc.add_tag("is_ttbar")
```

Comment:
Can you rename these tags to "ttbar" and "dy" (to be consistent with other tags already defined above).

Reply:
done